packages/clipboard/src/CopyCodeButton.tsx

```diff
@@ -8,19 +10,27 @@ import styles from './styles/CopyCodeButton.module.css';
 type CopyCodeButtonProps = ButtonHTMLAttributes<HTMLButtonElement> & {
   children: ReactNode;
   'data-test-id'?: string;
+  variant?: CopyToClipboardProps['variant'];
```

Comment:
Could we change this name to `kind` to be consistent with other components, and then add `default` to the kind as well since it's technically just another kind?